{{ cookiecutter.repo_name }}/.pre-commit-config.yaml

```diff
@@ -1,25 +1,25 @@
 default_language_version:
     python: python3
 repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.4.0
+    hooks:
+      - id: check-yaml
+      - id: end-of-file-fixer
+      - id: trailing-whitespace
   - repo: https://github.com/ambv/black
```

Comment:
I don't see the line length parameterization anywhere for black/flake? It'll start fighting with isort if the settings aren't consistent across.

Reply:
Wasn't viewing the full repo, only changes. `.flake8` and `pyproject.toml` are already present.

Reply:
We have a consistent line length configuration (115 symbols) for all three tools:
- [`flake8`](https://github.com/AbbyGi/scientific-python-cookiecutter/blob/670a54cd84ac43850037d7c1d13832e72b92d67e/%7B%7B%20cookiecutter.repo_name%20%7D%7D/.flake8#L10)
- [`isort`](https://github.com/AbbyGi/scientific-python-cookiecutter/blob/670a54cd84ac43850037d7c1d13832e72b92d67e/%7B%7B%20cookiecutter.repo_name%20%7D%7D/.isort.cfg#L2)
- [`black`](https://github.com/AbbyGi/scientific-python-cookiecutter/blob/670a54cd84ac43850037d7c1d13832e72b92d67e/%7B%7B%20cookiecutter.repo_name%20%7D%7D/pyproject.toml#L2)

I think we should port them to a single configuration file (`pyproject.toml`), but that may require some effort to figure out the right plugins and a consistent configuration via a single file.